pyaedt/icepak.py

```diff
@@ -1843,15 +1844,23 @@ def get_link_data(self, linkData):
             List containing the requested link data.
 
         """
-        if linkData[0] is None:
+
+        if "linkData" in kwargs:
+            warnings.warn(
+                "``linkData`` was deprecated in 0.6.43. Use ``links_data`` instead.",
```

Comment:
```suggestion
                "The ``linkData`` method was deprecated in 0.6.43. Use the ``links_data`` method instead.",
```
Google style rules say to follow a tagged literal with a noun.